src/frontend/src/app/[locale]/(auth)/dashboard/documents/page.tsx

```diff
@@ -0,0 +1,501 @@
+'use client';
```

Comment:
Use loading skeleton for the screen before fetching the file

https://nextjs.org/docs/app/api-reference/file-conventions/loading

Usually it says no files found and then it retrieves the files

We need to enable loading skeleton before loading Uploaded files and deleted files